manifests/copr.pp

```diff
@@ -0,0 +1,79 @@
+# Define: yum::copr
+#
+# This definition manages Copr (Cool Other Package Repo) repositories.
+#
+# Parameters:
+#   [*copr_repo*] - name of repository, defaults to title
+#   [*ensure*]    - specifies if repo should be enabled,
+#                   disabled or removed
+#
+# Actions:
+#
+# Requires:
+#   RPM based system
+#
+# Sample usage:
+#   yum::copr { 'copart/restic':
+#     ensure  => 'enabled',
+#   }
+#
+define yum::copr (
+  String                                 $copr_repo = $title,
```

Comment:
Variable `$repo` is proably enough since it's already in `yum::copr` type.

Reply:
Indeed, I changed that in the last force push :+1: .